macro-fullcalendar-ui/src/main/resources/Calendar/Macro.xml

```diff
@@ -288,6 +287,24 @@ or
 /************************************
  * Configure RequireJS to use jQuery-UI
  ************************************/
+// Moved require configuration to the Macro page because in case of
+// reloading the Chrome tab the resources will not be loaded.
```

Comment:
If you will look at the comment after, independent of the commit, you will not know from where the code was moved, it's not a relevant information. But instead you may say that this code needs to remain here because of the chrome problem. 
Also, the line should not  be wrapped before meeting the 120 characters limit.

Reply:
Thanks for the advice